# Define the goal
uint32 degrees  # in grad, wie weit gedreht werden soll 
float32 speed	#in grad pro sekunde
---
# Define the result
uint32 degrees_turned
bool canceled	#true, wenn abgebrochen, z.B. durch Wand
---
# Define a feedback message
float32 percent_complete
